bower.json

```diff
@@ -19,16 +19,30 @@
   "main": "iron-a11y-keys.html",
   "license": "http://polymer.github.io/LICENSE.txt",
   "dependencies": {
-    "polymer": "Polymer/polymer#^1.1.0",
-    "iron-a11y-keys-behavior": "PolymerElements/iron-a11y-keys-behavior#^1.0.0"
+    "polymer": "Polymer/polymer#^2",
```

Comment:
uhhh let's not make this change here? this hasn't been tagged, so the tests would fail anyway